Makefile

```diff
@@ -52,9 +52,27 @@ prettier: ## prettify the source code using prettier
 test: build test-unit lint ## launch all tests
 
 test-unit: ## launch unit tests
-	echo "Running unit tests...";
-	yarn -s test-unit;
+	@echo "Running unit tests...";
+	@yarn -s test-unit;
```

Comment:
FMI, what's the purpose of the '@' symbol?

Reply:
It ensure make does not print the command itself but just its result